docs/ruleset_functions/is_zone_mechanically_cooled.md

```diff
@@ -0,0 +1,30 @@
+# is_zone_mechanically_cooled
+**Schema Version:** 0.0.22  
+
+**Description:** determines whether a zone is mechanically cooled or not.  DOES NOT CHECK FOR TRANSFER AIR
```

Comment:
Since schema for transfer air is in flux, I would make the transfer air check a separate function.